release-tools/src/main/java/org/springframework/data/release/cli/ReleaseCommands.java

```diff
@@ -111,6 +104,28 @@ public void prepare(@CliOption(key = "", mandatory = true) TrainIteration iterat
 		git.commit(iteration, "Release version %s.");
 	}
 
+	@CliCommand(value = "repository open")
+	public void repositoryOpen(@CliOption(key = "", mandatory = true) TrainIteration iteration) {
+
+		// You don't "open" an artifactory release
+		if (!iteration.getIteration().isPublic()) {
```

Comment:
Let's invert the conditions for easier comprehension

Reply:
Done.